.changeset/neat-olives-reply.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/mongo-nav': minor
```

Comment:
@hswolff should this be a major bump, or are we adding additional functionality before we release v1

Reply:
In reviewing this, I'm not sure documentation is up to date for the exported components. Maybe the upgrade to v1.0.0 could include a pass of all docs?

Reply:
I can take a docs pass right now!